litestar/response/streaming.py

```diff
@@ -110,6 +116,25 @@ async def _listen_for_disconnect(self, cancel_scope: CancelScope, receive: Recei
             else:
                 await self._listen_for_disconnect(cancel_scope=cancel_scope, receive=receive)
 
+    async def _send_ping_event(self, send: Send) -> None:
```

Comment:
This shouldn't go into the base `ASGIStreamingResponse`, as pings don't really have a meaning there. This belongs strictly to SSE.

Also, there's an issue with this design, in that the ping task should gracefully stop if the SSE event generator is exhausted